fe/fe-core/src/main/java/com/starrocks/rpc/BrpcProxy.java

```diff
@@ -72,20 +72,24 @@ public static PBackendService getBackendService(TNetworkAddress address) {
         return getInstance().getBackendServiceImpl(address);
     }
 
-    public static LakeService getLakeService(TNetworkAddress address) {
+    public static LakeService getLakeService(TNetworkAddress address) throws RpcException {
         return getInstance().getLakeServiceImpl(address);
     }
 
-    public static LakeService getLakeService(String host, int port) {
+    public static LakeService getLakeService(String host, int port) throws RpcException {
         return getInstance().getLakeServiceImpl(new TNetworkAddress(host, port));
     }
 
     protected PBackendService getBackendServiceImpl(TNetworkAddress address) {
         return backendServiceMap.computeIfAbsent(address, this::createBackendService);
     }
 
-    protected LakeService getLakeServiceImpl(TNetworkAddress address) {
-        return lakeServiceMap.computeIfAbsent(address, this::createLakeService);
+    protected LakeService getLakeServiceImpl(TNetworkAddress address) throws RpcException {
+        try {
+            return lakeServiceMap.computeIfAbsent(address, this::createLakeService);
+        } catch (Exception e) {
+            throw new RpcException(e);
```

Comment:
original exception will catch the backend hosts and put it into exception message, may still need the information, if `e` doesn't contain the host info.

Reply:
Updated, PTAL